modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -250,75 +281,79 @@ List<CPOptionCategory> cpOptionCategories = cpContentHelper.getCPOptionCategorie
 				List<CPDefinitionSpecificationOptionValue> categorizedCPDefinitionSpecificationOptionValues = cpContentHelper.getCategorizedCPDefinitionSpecificationOptionValues(cpDefinitionId, cpOptionCategory.getCPOptionCategoryId());
 			%>
 
+			<table class="table table-sm table-striped">
```

Comment:
why this if is not wrapping the whole table?

Reply:
It is right, since in the design every specification field needs a table with the values. So it's one table for each specification